test/ansible-inventory/plugins/filter/test.py

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/python3
```

Comment:
Filter plugins shouldn't have a hashbang in them, they are imported directly, not executable.

Reply:
Without the `#!/usr/bin/python3` it will use the `#!/usr/bin/python` and fail in the tests.

Reply:
Just to register: After we spoke we agreed with moving forward with and then it will be checked with the ansible folks.